retirement_api/templates/claiming.html

```diff
@@ -560,28 +515,8 @@ <h3 class="lifestyle-question">{% trans questions.longlife.question %}</h3>
             </nav>
           </section>
         </div>
```

Comment:
I think we can remove this whole `div.nemo`. Looks like it just contains an extra copy of the subnav now that the subnav has been moved to the base template.